SConstruct

```diff
@@ -406,14 +405,14 @@ def partition(c):
 @w.add_nest(label_func=lambda d: d['id'], metadata=lambda c, d: {'annotation': 'elided', 'naive_probabilities': 'elided'})
 def cluster(c):
     part = c['partition']
-    naive_probabilities = None
-    seed_cluster_annotation = part.get('seed_cluster_annotation')
-    if seed_cluster_annotation:
-        naive_probabilities = get_alt_naive_probabilities(seed_cluster_annotation)
+    seed_cluster_annotation = part['seed_cluster_annotation']
+    unique_ids = ':'.join(seed_cluster_annotation['unique_ids'])
+    naive_probabilities = get_alt_naive_probabilities(seed_cluster_annotation)
     return [{'id': 'seed-cluster',
              'seed_name': c['seed']['id'],
              'size': part['seed_cluster_size'],
              'annotation': seed_cluster_annotation,
+             'unique_ids': unique_ids,
```

Comment:
Be careful; This one can get pretty big, and I vaguely recall having to elide it for some reason.

Reply:
On second thought, this may only be an issue for unseeded clusters.

Reply:
Good point. I don't think we need to write linearham input files for unseeded clusters right now, so hopefully it's not an issue?

Regardless, I think the reason I used unique ids strings for this was because I wasn't sure if [the logic in this function](https://github.com/matsengrp/cft/blob/276-linearham-integration/bin/process_partis.py#L277) would hold up in this case, but I don't see why it wouldn't. That same function is used to select the seed cluster initially (when bin/process_partis.py is called from SConstruct) and then it happens to get subsetted/downsampled after that (and pruned etc), but since we are passing a file containing the uids we want from said cluster, all that should be preserved as far as I can tell. So I can change it to not need the unique ids and select the cluster using https://github.com/matsengrp/cft/blob/276-linearham-integration/bin/process_partis.py#L289 if we have any concerns about the unique_ids slowing things down (or arguably just for consistency's sake).

Reply:
Cool; Thanks for clarifying. Your call on the right approach.